### DISCUSSION TITLE
Trakt is switching to TMDB for TV show metadata

### DISCUSSION THREAD

**Discussion by rectifyer** [trakt/trakt-api#250] [Announcements]
Here's the blog post we just sent out announcing Trakt is switching to TMDB for TV show metadata.
https://trakt.tv/b/tmdb

As a developer, this doesn't really change how you use the Trakt API. However, if you rely solely on TVDB IDs, I would recommend having fall backs to use Trakt, TMDB, and IMDB IDs as well.

TMDB has the concept of "external IDs" and when editing TMDB, we highly encourage adding the TVDB ID in that section. This would be on the show and episode level (I don't believe TMDB has support at the season level). Having as many external IDs as possible helps with matching.

Please post any questions and we can start the discussion.

**Comment by rlaphoenix** [comment]
Thanks for doing this!

One quick question, how are you planning to actually take care of the differences/unmatching data between TVDB and TMDB? Will staff members or moderators be editing TMDB's public api data to accommodate changes?

**Reply by rectifyer** [reply]
We’ll largely be relying on the community to help add and fix data at TMDB. We’ll do some ourselves too, but with only 2 of us it’s tough to devote a ton of our time to it. We’re also considering hiring a dedicated support person in the future and they would help update TMDB data along with the main support responsibilities.

**Comment by xstasy** [comment]
A lot of apps have relied on tvdb id's for a long time, is this data just going to disappear now?
